interview-questions.md

```diff
@@ -7,53 +7,53 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Rails follows an MVC pattern. What does that mean?
 
-  Your answer:
+  Your answer: I have heard of this before and I believe that MVC stands for model, View and control. Do you mean what it means is the ability to talk to databases and how we view them according to rails.
 
-  Researched answer:
+  Researched answer: So I did a little of research and I found out that MVC means that in rails when you look at a database  the request first come to the controller and then the controller finds an appropriate view and interacts with model which then interacts with database and send response to back to you.
 
 
 
 2. What is a relational database? Are there other kinds of databases?
 
-  Your answer:
+  Your answer: Thanks for white boarding this week with Sarah and me taking better notes she asked Raul this question so I remembered that relational database is a type of database that stores and provides access to data points that are related to one another. 
 
-  Researched answer:
+  Researched answer: During my research I reflected back on the syllabus as well and it made it clear that a good example of a relational database is like Cohart versus students both of those tables and database have stuff in common call heart could have many students why a student is only belongs to one cohort.
 
 
 
 3. What is object-relational mapping?
 
-  Your answer:
+  Your answer: Object related mapping Has to do with object oriented programming. Object oriented programming is like the blueprint of programming and object relational mapping is how you read the blueprint. 
 
-  Researched answer:
+  Researched answer: Object–relational mapping is a programming technique for converting data between incompatible type systems using object-oriented programming languages. As you can see I was correct in assuming that it had to do with the object oriented programming.
 
 
 
 4. What is a gem?
 
-  Your answer:
+  Your answer: Gems make your Rails app more functional. Which means it makes the app do more then it would be able to without the gym and you can install as many gems as you want.
 
-  Researched answer:
+  Researched answer: So I found out that I was half right except for it deals with functions and methods that other people have written in Ruby and then turns them into gym so where other people can use it without having to write this in functions or methods over and over again.
```

Comment:
Yea, they're bunches of code that others have written that we can install into our projects.